ngi_reports/utils/entities.py

```diff
@@ -74,79 +78,90 @@ def set_lane_info(self, to_set, key, lane_info, reads, as_million=False):
-                            }
-        self.application = ''
+        self.accredited = {
+            "library_preparation": "N/A",
+            "data_processing": "N/A",
+            "sequencing": "N/A",
+            "data_analysis": "N/A",
+        }
+        self.application = ""
         self.best_practice = False
-        self.cluster = ''
-        self.contact = ''
-        self.dates = { 'order_received': None,
-                        'open_date':  None,
-                        'first_initial_qc_start_date': None,
-                        'contract_received': None,
-                        'samples_received': None,
-                        'queue_date': None,
-                        'all_samples_sequenced': None
-                    }
+        self.cluster = ""
+        self.contact = ""
+        self.dates = {
+            "order_received": None,
+            "open_date": None,
+            "first_initial_qc_start_date": None,
+            "contract_received": None,
+            "samples_received": None,
+            "queue_date": None,
+            "all_samples_sequenced": None,
+        }
         self.is_finished_lib = False
         self.is_hiseqx = False
-        self.library_construction  = ''
-        self.missing_fc  = False
-        self.ngi_facility = ''
-        self.ngi_name = ''
-        self.samples_unit  = '#reads'
+        self.library_construction = ""
+        self.missing_fc = False
+        self.ngi_facility = ""
+        self.ngi_name = ""
+        self.samples_unit = "#reads"
         self.num_samples = 0
         self.num_lanes = 0
-        self.ngi_id = ''
-        self.reference = { 'genome': None,
-                            'organism': None }
-        self.report_date = ''
-        self.sequencing_setup = ''
+        self.ngi_id = ""
+        self.reference = {"genome": None, "organism": None}
+        self.report_date = ""
+        self.sequencing_setup = ""
         self.skip_fastq = False
-        self.user_ID = ''
+        self.user_ID = ""
 
     def populate(self, log, organism_names, **kwargs):
-
-        project = kwargs.get('project', '')
+        project = kwargs.get("project", "")
         if not project:
-            log.error('A project must be provided, so not proceeding.')
-            sys.exit('A project was not provided, stopping execution...')
-        self.skip_fastq = kwargs.get('skip_fastq')
-        self.cluster = kwargs.get('cluster')
+            log.error("A project must be provided, so not proceeding.")
+            sys.exit("A project was not provided, stopping execution...")
+        self.skip_fastq = kwargs.get("skip_fastq")
+        self.cluster = kwargs.get("cluster")
 
         pcon = statusdb.ProjectSummaryConnection()
-        assert pcon, 'Could not connect to {} database in StatusDB'.format('project')
+        assert pcon, "Could not connect to {} database in StatusDB".format("project")
 
-        if re.match('^P\d+$', project):
+        if re.match("^P\d+$", project):
```

Comment:
```suggestion
        if re.match(r"^P\d+$", project):
```
This shows a warning sometimes